src/crimson/osd/object_context_loader.cc

```diff
@@ -47,27 +47,33 @@ using crimson::common::local_conf;
           crimson::ct_error::enoent::make()
         };
       }
+      auto resolved_oid = resolve_oid(head->get_head_ss(), oid);
+      if (!resolved_oid) {
+        ERRORDPP("clone {} not found", dpp, oid);
+        return load_obc_iertr::future<>{
+          crimson::ct_error::enoent::make()
+        };
+      }
+      if (resolved_oid->is_head()) {
+        // See resolve_oid
+        return std::move(func)(head, head);
+      }
```

Comment:
During our teuthology test, `resolve_oid` here will lead to the failure of recovering clone objects to replica osds, when the `SnapSet::clone_snaps` is like `f:[c, b]`. In this case, snapshot `f` has been removed, if we need to recover clone object `oid:f` to replica osds, the `resolve_oid` will fail.